node/parallel-dev/src/chain_spec.rs

```diff
@@ -214,9 +217,9 @@ fn testnet_genesis(
                 (CurrencyId::xKSM, Ratio::from_percent(90)),
             ],
             liquidation_incentive: vec![
-                (CurrencyId::KSM, Ratio::from_percent(90)),
-                (CurrencyId::USDT, Ratio::from_percent(90)),
-                (CurrencyId::xKSM, Ratio::from_percent(90)),
+                (CurrencyId::KSM, DEFAULT_LIQUIDATION_INCENTIVE_PCT),
```

Comment:
You have changed the value from 90% to 110%, but haven't change the place that uses this storage, the logic should be different when using the new value, can you check this?

Reply:
before: collateral_value * incentive(90%) = repay_value
expect: collateral_value = repay_value * incentive(110%)